src/test/java/org/isf/medicalstock/test/Tests.java

```diff
@@ -1027,22 +1083,22 @@ public void testMovementToString() throws Exception {
 		int code = _setupTestMovement(false);
 		Movement movement = movementIoOperationRepository.findOne(code);
 		// TODO: if resource bundles are available this string test needs to change
-		assertThat(movement.toString()).isEqualTo("angal.medicalstock.medical:TestDescriptionangal.medicalstock.type:TestDescriptionangal.common.quantity:10");
+		assertThat(movement).hasToString("angal.medicalstock.medical:TestDescriptionangal.medicalstock.type:TestDescriptionangal.common.quantity:10");
 	}
 
 	@Test
 	public void testMovementEquals() throws Exception {
 		int code = _setupTestMovement(false);
 		Movement movement = movementIoOperationRepository.findOne(code);
 		assertThat(movement.equals(movement)).isTrue();
-		assertThat(movement.equals(null)).isFalse();
-		assertThat(movement.equals("someString")).isFalse();
+		assertThat(movement).isNotEqualTo(null);
```

Comment:
Just wondering in case I missed a previous discussion, have you agreed anytime before that this line (and equivalents) supposed to be more readable than `assertNotNull(movement)` ?

Reply:
I was trying to test the `equals()` method and I wasn't sure what `assertNotNull()` used under the covers but I did know what `isNotEqualTo()` used and that was the method under test.

Reply:
OK. I can see your point and for sure:  

     assertThat(movement).isNotEqualTo(null);

looks much better than:

    assertThat(movement.equals(null)).isFalse();

but, remembering old discussions about why the adoption of assertThat from assertj and how it would suppose to be beneficial, this is one of the cases where I don't think any of both beat the simplicity/readability of a:

     assertNotNull(movement) 

It is just my personal opinion :)

Reply:
am I wrong or `assertNotNull` has never been used up to now?

Reply:
Probably not but it is a very valid method

Reply:
I'm sure of that, so we want to introduce it or we merge as is this PR?

Reply:
I would merge it

Reply:
Alright, that's why I was verify the change's consistency...